packages/frontend/core/src/components/affine/onboarding/articles/index.tsx

```diff
@@ -244,8 +244,7 @@ const states: Partial<Record<ArticleId, EdgelessSwitchState>> = {
 
 export const articles: Record<ArticleId, ArticleOption> = ids.reduce(
   (acc, id) => {
-    return {
-      ...acc,
+    return Object.assign(acc, {
```

Comment:
This seems different code logic, change the origin object. Is it correct?

Reply:
the original object is created by the second param `{} as Record<ArticleId, ArticleOption>`, so it's totally okay here

Reply:
`{...acc, ...xxx}` is considered as best practice now?

Reply:
Only if it was in `Array#reduce`, it's covered by the `oxlint -D perf`

Reply:
Cool! Now I get it,